rescheduler/Makefile

```diff
@@ -1,25 +1,65 @@
-all: build
 
 FLAGS =
 ENVVAR = GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 REGISTRY = gcr.io/google-containers
 TAG = v0.3.1
+ARCH ?= $(shell go env GOARCH)
+ALL_ARCH = amd64 arm arm64 ppc64le
+
+
+IMAGE = $(REGISTRY)/rescheduler
+MULTI_ARCH_IMG = $(IMAGE)-$(ARCH)
+
+BASEIMAGE ?= busybox:latest
+
+ifeq ($(ARCH),arm)
+        BASEIMAGE=armhf/busybox:latest
+endif
+ifeq ($(ARCH),arm64)
+        BASEIMAGE=arm64v8/busybox:latest
+endif
+ifeq ($(ARCH),ppc64le)
+        BASEIMAGE=ppc64le/busybox:latest
+endif
 
 deps:
 	go get github.com/tools/godep
 
 build: clean deps
-	$(ENVVAR) godep go build ./...
-	$(ENVVAR) godep go build -o rescheduler
+	GOOS=linux GOARCH=$(ARCH) CGO_ENABLED=0 godep go build ./...
```

Comment:
is godep really needed anymore?

Reply:
I do agree, if godep is not required then we can remove godep dependencies.

Reply:
Godep was there earlier. Reran the build after removing it, was able to run it successfully, guess we don't need it, have removed it!